### DISCUSSION TITLE
【疑似Bug】两个 Adjust dodge 配置，在 4.1.40 版本正常渲染，>= 4.1.41 版本上渲染为空

### DISCUSSION THREAD

**Discussion by sfrog** [antvis/G2#5080] [Q&A]
Case 1，4.1.40 版本正常渲染：
https://stackblitz.com/edit/react-ycwskd?file=src%2Findex.js,package.json

Case 2，4.1.41 版本渲染异常：
https://stackblitz.com/edit/react-yuazto?file=src%2Findex.js,package.json

求各位帮我看看，这种使用场景是需要支持的吧？从 4.1.41 版本开始就不能用了。

**Comment by hustcc** [comment]
可能是某个 pr 带来的 Adjust bug，其实这种情况，更建议在数据处理阶段，把两个字段合并成一个，然后用这个合并之后的字段作为 dodge。

**Reply by sfrog** [reply]
谢谢回复
我们的业务场景中，这两个 dodge 是不同层面的：先按第一个层面 dodge，柱子之间间隔距离稍大；再按第二个层面 dodge，间隔距离较小。所以我们用了两个不用的 dodge
